test/src/test/java/org/corfudb/integration/StreamingIT.java

```diff
@@ -457,6 +457,70 @@ public void testMoreSubscribersAndSlowSubscribers() throws Exception {
         assertThat(shutdownCorfuServer(corfuServer)).isTrue();
     }
 
+    /**
+     * Test the case where a table is empty at the time of subscription, and that it is able
+     * to receive deltas once updates to the table happen.
+     */
+    @Test
+    @SuppressWarnings("checkstyle:magicnumber")
+    public void testSubscriberOnEmptyTables() throws Exception {
+        // Run a corfu server.
+        Process corfuServer = runSinglePersistentServer(corfuSingleNodeHost, corfuStringNodePort);
+
+        // Start a Corfu runtime.
+        runtime = createRuntime(singleNodeEndpoint);
```

Comment:
I don't feel like this case deserves a separate unit test, can it be merged with `testStreamingSingleTable` ?

Reply:
I would prefer to leave it as a separate test cause when a test is overloaded we tend to think the case is not covered.  Typically we (or at least me) just read the test names or quickly do a ctrl+find on key names...